cfme/common/vm_views.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
```

Comment:
Actually, what I'm doing with provider views. I put almost all of them to provider_views file until all providers are moved to widgetastic. When All providers are converted. I'll move common views to common/provider.py and the rest will be moved to appropriate provider files.